invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -123,51 +136,56 @@ def user_dashboard_request_view(request, **kwargs):
 @pass_community(serialize=True)
 def community_dashboard_request_view(request, community, community_ui, **kwargs):
     """Community dashboard requests details view."""
-    request_type = request["type"]
-
     avatar = current_user_resources.users_service.links_item_tpl.expand(
         g.identity, current_user
     )["avatar"]
 
+    request_type = request["type"]
+
     is_draft_submission = request_type == CommunitySubmission.type_id
-    is_invitation = request_type == CommunityInvitation.type_id
+    is_record_inclusion = request_type == CommunityInclusion.type_id
+    is_member_invitation = request_type == CommunityInvitation.type_id
     request_is_accepted = request["status"] == AcceptAction.status_to
-    if is_draft_submission:
+
+    if is_draft_submission or is_record_inclusion:
```

Comment:
not in the context of this PR, feel free to ignore for now.
we will have to probably refactor these conditional statements at some point, especially when we have more request types